OutboundTests/OBConfig+UnitTests.m

```diff
@@ -10,21 +10,6 @@
 
 @implementation OBConfig (UnitTests)
 
-//+ (void)getSdkConfigWithCompletion:(void (^)(OBConfig *config))completion {
```

Comment:
Any reason for keeping this?

Reply:
No, it was commented out as a step on the way and kept for reference. Didn't remove it after progressing with the tests.